<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>fTools About</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="3" row="0" column="0" >
    <widget class="QLabel" name="fToolsLogo" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="font" >
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>fTools</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Version x.x-xxxxxx</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QTextEdit" name="textEdit" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="0" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="0" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="html" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" >
    <widget class="QPushButton" name="btnHelp" >
     <property name="text" >
      <string>Help</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="btnWeb" >
     <property name="text" >
      <string>Web</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QPushButton" name="pushButton" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel" >
     <x>267</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
